### ISSUE TITLE
QRIS payment for Xendit

### ISSUE THREAD

**Issue by bxcodec** [imrenagi/go-payment#12]
Xendit already support for QRIS

**Comment by bxcodec** [comment]
Will try to explore by this weekend if you busy 🙏

**Comment by imrenagi** [comment]
Yeah, I see it on the docs, but still thinking how the QRIS QR Code should be visualised after I got the text from xendit. Should it be a page view provided by the backend, or should it do something else like generating image etc. 

Sure. Looking forward to it, mas bro!
